src/reference/asciidoc/whats-new.adoc

```diff
@@ -86,6 +86,11 @@ Previously, the consumer tags for asynchronous consumers were generated by the b
 With this release, it is now possible to supply a naming strategy to the listener container.
 See <<consumerTags>>.
 
+===== RabbitTemplate Blocking Receive Methods
+
+The `RabbitTemplate` now supports blocking in `receive()` and `convertAndReceive` methods.
```

Comment:
Or brackets for both cases, or no one. Fortunately you have "methods" word, so no need to worry about brackets.
Nit-pick, of course. But it would be better always be on our toes do not bring more bad errors :smile: